### PR TITLE
[Project Submission] Luxurify

### PR DESCRIPTION
**Pull request by nnluukhtn**
Adding Luxurify to Projects

### CODE REVIEW

File: projects/luxurify.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+### Build for Collector
+- Convenient marketplace.
+- Fast, Secure, Trusted place to buy Authentic Luxury Watches.
+
+
+## Project Status
+<!--brainstorming, fundraising, under development, beta, shipped, etc-->
+Under Development
+
+## Previews
```
Comment (autonome) [inline projects/luxurify.md:38]: Can you add screenshots?

File: projects/luxurify.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+## Target Audience
+<!--Describe who will be your project's users-->
+Watch's brands, owners and collectors
+
+## Rough estimated user base (if applicable)
+<!--How many users do you have right now?-->
+N/A
+
+## Github repo
```
Comment (autonome) [inline projects/luxurify.md:50]: Hi, please add a license file and license as MIT

**Review by autonome — CHANGES_REQUESTED** [review]
Thanks for joining! a couple of changes needed

**Review by autonome — CHANGES_REQUESTED** [review]
Looking good, just a couple of changes noted in the review!

File: projects/luxurify.md (hunk 1 above)
Comment (nnluukhtn) [reply projects/luxurify.md:38]: Added @autonome

File: projects/luxurify.md (hunk 2 above)
Comment (nnluukhtn) [reply projects/luxurify.md:51]: Added MIT license @autonome

**Comment by autonome** [comment]
Looks good, thanks!

**Pull request merged**